server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -263,7 +263,7 @@ export interface IDb {
 
 	on(event: IDbEvents, listener: (...args: any[]) => void);
 
-	collection<T>(name: string): ICollection<T>;
+	collection<T>(name: string, dbName?: string): ICollection<T>;
```

Comment:
Does the associated implementation of this interface need to change to match this interface update?

Reply:
Oh, I guess it already takes that parameter into account:
https://github.com/microsoft/FluidFramework/blob/500e404c31c9ce26da4696c9a2f45fb624be2ad3/server/routerlicious/packages/services/src/mongodb.ts#L514